packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/edit/edit.html

```diff
@@ -0,0 +1,262 @@
+<oui-modal
+    data-ng-form="editNamespaceForm"
+    data-on-dismiss="$ctrl.cancel()"
+    data-primary-label="{{:: 'pci_databases_namespaces_edit_button_label' | translate}}"
+    data-primary-disabled="editNamespaceForm.$invalid"
+    data-primary-action="$ctrl.edit()"
+    data-secondary-action="$ctrl.cancel()"
+    data-secondary-label="{{:: 'pci_databases_namespaces_edit_cancel_button_label' | translate}}"
+    data-loading="$ctrl.processing"
+    data-heading="{{:: 'pci_databases_namespaces_edit_title' | translate}}"
+>
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_edit_name_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <input
+                    class="oui-input"
+                    type="text"
+                    id="namespace_name"
+                    name="namespace_name"
+                    ng-model="$ctrl.model.name"
+                    ng-pattern="$ctrl.FORM_RULES.name.pattern"
+                    ng-maxlength="$ctrl.FORM_RULES.name.max"
+                    ng-minlength="$ctrl.FORM_RULES.name.min"
+                    placeholder="{{:: 'pci_databases_namespaces_edit_name_placeholder' | translate }}"
+                    disabled
+                    required
+                />
+            </oui-field>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_edit_type_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    aria-label="Help"
```

Comment:
**todo**: This needs a translation

Reply:
Hi @anooparveti , this has already been discussed before, please see Deena's comment on ticket PUD-1256.